bahmnicore-ui/pom.xml

```diff
@@ -75,18 +77,6 @@
             <artifactId>reference-data-api</artifactId>
             <version>${project.version}</version>
         </dependency>
-        <dependency>
```

Comment:
any specific reason for excluding openmrs-api?

Reply:
We have duplicate openmrs-api dependency added in the pom.xml. We see these warnings when we run `mvn install` command.